### ISSUE TITLE
CI fails due to mismatched ocamlformat versions

### ISSUE THREAD

**Issue by zhengqunkoo** [hipsleek/hipsleek#48]
The installed ocamlformat is version 0.14.3, but the version specified at https://github.com/hipsleek/hipsleek/blob/master/.ocamlformat is 0.14.2.

CI continues to succeed at this repository, but fails at https://github.com/zhengqunkoo/hipsleek. This could be due to installing from the CI cache instead of from `rake dependencies:install`.

The solution is likely to bump the version number, and stop installing from CI cache. This should not be too harmful, as CI cache does not seem to provide much speedup.

**Comment by benedictleejh** [comment]
`ocamlformat` has extremely strict versioned requirements in format specification. Because of that, it's probably best to just fix the `ocamlformat` version directly, unless you want to chase versions. I didn't so I just fixed the installed version: benedictleejh/hipsleek@95091dccfff364abc88a7a3dfb3a8f01039239dc. That will also fix the version used in `rake dependencies:install`.

The success of the CI here is because the VMs used for testing already have `ocamlformat` installed, so it doesn't get reinstalled at every test run.

**Comment by zhengqunkoo** [comment]
> The success of the CI here is because the VMs used for testing already have `ocamlformat` installed, so it doesn't get reinstalled at every test run.
Yes, that's due to `$CACHE_NAME` being set in `.travis.yml`. Otherwise Travis CI would reinstall on every job.

The solution at https://github.com/benedictleejh/hipsleek/commit/95091dccfff364abc88a7a3dfb3a8f01039239dc looks good to me. For consistency, either `Makefile` should have a similar fixing of the package version, or `Makefile` should be deprecated.
